lib/ansible/modules/windows/win_copy.py

```diff
@@ -44,59 +33,89 @@
     - Remote absolute path where the file should be copied to. If src is a
       directory, this must be a directory too.
```

Comment:
How about adding something like "and it will be created (with the parent directories) if it (or they) doesn't exist"? Otherwise, users might think they have to ensure that `dest` directory exists before.